### PR TITLE
Consuming the API for application login

### LINKED ISSUE #10: Login validation with Ruby API
# REQUIREMENTS
The user can do login with Ruby API

# Acceptance Criteria
List all required acceptance criteria
- [ ] Do login funtionallity
- [ ] Validate with the email or password is incorrect

### PR DESCRIPTION
**Pull request by hearagmu**
# Description
Consuming the API for application login, using AXIOS library.
 
 # Testing
Before we tried with a Postman, to evaluate the behavior of the API.
image
  
  ## Screenshots 
2021-03-22_09h13_42

### CODE REVIEW

**Review by mariam-rivera — DISMISSED** [review]

**Review by vmagana3 — DISMISSED** [review]
Good work héctor!!

File: COVID19/src/components/Input.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ import { View, TextInput } from 'react-native';
...
-const Input = ({placeholderTxt, iconName}) => {
+const Input = ({placeholderTxt, iconName, onCh}) => {
     return(
         <View style={styles.inputContainer}>
             <Icon name={iconName} color="gray" size={25} style={{marginStart: 15, alignSelf:'center'}}/>
             <TextInput                 
                 placeholder={placeholderTxt}
                 secureTextEntry={placeholderTxt === 'Password' && true}
                 placeholderTextColor="gray"
+                onChange={onCh}
```
Comment (LeyaimMagma) [inline COVID19/src/components/Input.js:14]: bad prop name, on change what?

File: COVID19/src/screens/Login.js [hunk 2]
```diff
@@ -1,25 +1,61 @@
...
+  const [formData, setFormData] = useState(defaultValue());
+  const onChange = (e, type) => {
+    setFormData({...formData, [type]: e.nativeEvent.text});
+  };
+  const putLoginOk = () => {
+    if (
+      (formData.usuario || formData.contrasenia) &&
+      (formData.usuario !== '' || formData.contrasenia !== '')
+    ) {
+      const url = 'https://hidden-cliffs-21927.herokuapp.com/user/sign_in?user[email]='+formData.usuario+'&user[password]='+formData.contrasenia;
```
Comment (LeyaimMagma) [inline COVID19/src/screens/Login.js:20]: Add this URL on a constant files

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: COVID19/src/components/Input.js (hunk 1 above)
Comment (hearagmu) [reply COVID19/src/components/Input.js]: fixed!

File: COVID19/src/screens/Login.js (hunk 2 above)
Comment (hearagmu) [reply COVID19/src/screens/Login.js]: fixed!

**Review by LeyaimMagma — APPROVED** [review]

**Review by jjcr11 — APPROVED** [review]
:+1:

**Pull request merged**